.github/workflows/test.yaml

```diff
@@ -18,9 +18,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@stable
```

Comment:
Why are these removed from the workflow?

Reply:
rust is included by default in github runners

Reply:
Go Github!

Reply:
I added a `rust-toolchain.toml`, when this file is present, GitHub CI will automatically install and pin the specified Rust version without need for an action